src/components/grid-aware/Footer/SubscriptionBlock/SubscriptionBlock.module.css

```diff
@@ -0,0 +1,59 @@
+/* Background bleed */
+
+.bleedWrapper {
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+/* Top Row */
+
+.subscriptionBlock {
+  background-color: var(--color-black);
+  display: flex;
+  justify-content: center;
+  padding: 35px;
+}
+
+.subscribeContainer {
+  align-items: baseline;
+  display: flex;
+}
+
+.title {
+  color: var(--color-white);
+  font: var(--font-subtitle);
+  margin-right: 97px;
+  width: 263px;
+}
+
+.formInput {
+  display: flex;
+}
+
+.inputText {
+  width: 291px;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    display: block;
+  }
+
+  .subscribeContainer {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .title {
+    margin-right: 0;
+  }
+
+  .formInput {
+    margin-top: 22px;
+  }
+
+  .inputText {
+    width: 155px;
```

Comment:
I think it'd be better to let this grow flexibly in smaller screen sizes than to set a fixed size. Space is very limited in mobile, and we'll want as much space as we can get.

Can you just set this to `auto`? Alternatively, in the non-mobile view, you can change it from `width: 291px` to `max-width: 291px`.

Reply:
I can't let the text-box get too small so I have to set a min-width
    width: auto;
    min-width: 155px;